### DISCUSSION TITLE
Nginx proxy multi tenant URL

### DISCUSSION THREAD

**Discussion by eableson** [minio/minio#14371] [Q&A] [answered]
I'm trying to setup a multi-tenant minio installation, but want to keep a single front-end hostname instead of individual host names per tenant. IE, replacing the traditional mode of:
tenant1.s3.example.com -> cluster:9001
tenant2.s3.example.com -> cluster:9002
tenant1.console.example.com -> cluster:10001
tenant2.console.example.com -> cluster:10002

with 
s3.example.com/tenant1 -> cluster:9001
s3.example.com/tenant2 -> cluster:9002
console.example.com/tenant1 -> cluster:10001
console.example.com/tenant2 -> cluster:10002

I've fiddled with the nginx configuration and with a rewrite command in the location block, I can get the s3 portion working fine. But I've tried every configuration option I can think of and the console *always* redirects back to the root, as in:

console.example.com/login

It completely ignores the rewrite options, and I can't put a trailing path in the MINIO_BROWSER_REDIRECT_URL as this is refused.

Ideas?

**Comment by harshavardhana** [comment, marked as answer]
Trailing paths are not supported for Console UI

**Reply by eableson** [reply]
Thanks for the quick reply - I'll stop wasting time on trying to get that to work then. But I will put in a note for this as a feature request.

Cheers,

Erik

**Reply by harshavardhana** [reply]
> Thanks for the quick reply - I'll stop wasting time on trying to get that to work then. But I will put in a note for this as a feature request.

👍🏽
